public/app/percona/settings/Settings.tsx

```diff
@@ -30,6 +30,8 @@ export const SettingsPanel: FC = () => {
     refresh?: boolean,
     onError = () => {}
   ) => {
+    const testEmail = body.email_alerting_settings.test_email || '';
+    body.email_alerting_settings.test_email = undefined;
```

Comment:
Why do we need to remove the test_email from the body?

Reply:
Yeah, this one is hard to follow, I'll add a comment there.
The thing is: when you submit the form, you don't want the values there to vanish after the re-render. All other fields stay there because we use the values coming from the API response. But for the test email that wouldn't happen, as that field is not part of the request. So, the value is being sent all the way up to the parent component (`Settings.tsx`) so that it can be later on sent down with the payload again. But since we don't want to send this field in the request payload, I'm just erasing it here.

Reply:
Thank you for the explanation!